README.md

```diff
@@ -16,8 +16,13 @@ So far we tested Northstar on
 * 64-bit ARM
 * x86_64
 
-### Missing from implementation
+### Everything that is checked is implemented
 
+- [x] On-the-fly verification of container content
+- [x] Process supervision: memory control
+- [x] Process supervision: cpu control
```

Comment:
[CGroups](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt) configuration support (memory, CPU)

Reply:
same thing...I'd say the memory and cpu control is the feature...cgroups are just what we use to implement it

Reply:
Yes, but still worth to mention that it makes use of cgroups.